glue/core/qt/roi.py

```diff
@@ -123,8 +123,8 @@ def __init__(self, axes):
 
 class QtPolygonalROI(QtROI, roi.MplPolygonalROI):
 
-    def __init__(self, axes):
-        roi.MplPolygonalROI.__init__(self, axes)
+    def __init__(self, axes, _roi=None):
+        roi.MplPolygonalROI.__init__(self, axes, roi=_roi)
```

Comment:
Why the ``_`` in the keyword argument name?

Reply:
It's because `roi` was already being used as a module name (`from glue.core import roi`, also see the line immediately below). It might make more sense to change the import, though,  to be able to use `roi` as the keyword argument?